v1/src/test/java/com/google/cloud/teleport/templates/TextImportPipelineIT.java

```diff
@@ -130,6 +130,7 @@ public void testImportCsv() throws IOException {
             .addParameter("instanceId", googleSqlResourceManager.getInstanceId())
             .addParameter("databaseId", googleSqlResourceManager.getDatabaseId())
             .addParameter("spannerProjectId", PROJECT)
+            .addParameter("spannerHost", spannerHost)
```

Comment:
Hm, why did we need this? Did something change which now requires the host to be specified?

Reply:
The `google_cloud_spanner_version` has changed from 6.88.0 to 6.89.0.
In the older version there is a `SpannerOptions.CLOUD_SPANNER_HOST_PATTERN.matcher(host).matches()` which causes NPE if host is empty.
I think instead of this change I should just update version to the 6.89.0 for the DataflowTemplates as well.

Reply:
@damccorm I set 6.89.0, 
Could you please approve workflows

Reply:
Sure - I think the problem is actually https://github.com/apache/beam/pull/34489 though